bin/http-server

```diff
@@ -24,6 +24,7 @@ if (argv.h || argv.help) {
     '  -i           Display autoIndex [true]',
     '  -e --ext     Default file extension if none supplied [none]',
     '  -s --silent  Suppress log messages from output',
+    '  --contentType Define the content-type default value',
```

Comment:
I know this is a tiny nitpick, but could you align "Define the..." with "Enable CORS..." below?